lib/events.js

```diff
@@ -322,7 +322,7 @@ EventEmitter.prototype.emit = function emit(type, ...args) {
     }
   } else {
     const len = handler.length;
-    const listeners = arrayClone(handler, len);
+    const listeners = arrayClone(handler);
```

Comment:
we don't actually have to clone if `len < 2`, another possible optimization?

Reply:
I'm not sure what you mean, if we hit this branch then we already know `handler.length >= 2`.

Reply:
Ah, sorry. Didn't look at the full context.